src/ontology/cl-edit.owl

```diff
@@ -15517,7 +15519,7 @@ AnnotationAssertion(oboInOwl:creation_date obo:CL_0002206 "2010-09-02T02:38:36Z"
 AnnotationAssertion(oboInOwl:hasAlternativeId obo:CL_0002206 "CL:1000390"^^xsd:string)
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0002206 "FMA:263228"^^xsd:string)
 AnnotationAssertion(rdfs:label obo:CL_0002206 "brush cell of terminal bronchiole"^^xsd:string)
-EquivalentClasses(obo:CL_0002206 ObjectIntersectionOf(obo:CL_0002075 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001958)))
+EquivalentClasses(obo:CL_0002206 ObjectIntersectionOf(obo:CL_0002204 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001958)))
```

Comment:
'brush cell of lobular bronchiole' is an inferred parent class.

My understanding is that lobular bronchiole and terminal bronchiole are distinct anatomical structures, so this inference does not seem correct.

Reply:
The inference comes from 'epithelium of bronchiole' is an exact synonym of 'epithelium of lobular bronchiole'
<img width="1359" alt="Screenshot 2023-02-06 at 14 51 39" src="https://user-images.githubusercontent.com/94959119/217004622-8eed1a50-fe07-4247-ac7d-25728e35ea2a.png">

My suggestion is:

- [ ] Create a 'lobular bronchiole epithelium' term separate from ''epithelium of bronchiole''

Reply:
This seems reasonable if the synonyms are also removed and 'lobular bronchiole' becomes a new Uberon class.

Reply:
I don't think annotated synonyms impact inferences.

Reply:
'brush cell of lobular bronchiole' should be `'brush cell' and ('part of' some 'epithelium of lobular bronchiole')`. However, as 'epithelium of lobular bronchiole' is an exact synonym of 'epithelium of bronchiole', it is inferred that 'brush cell of terminal bronchiole' is a subclass of 'brush cell of lobular bronchiole'

Reply:
bronchiole and lobular bronchiole have been used as synonyms in seven terms (some are repeated):
<img width="1359" alt="Screenshot 2023-02-06 at 16 15 02" src="https://user-images.githubusercontent.com/94959119/217025186-55f1514f-3738-475b-8210-4d1efa15d140.png">

Therefore, I will expand the creation of new terms.